tests/acceptance/seller/InternalTransferCest.php

```diff
@@ -0,0 +1,84 @@
+<?php 
+
+namespace hipanel\modules\finance\tests\acceptance\seller;
+
+use hipanel\helpers\Url;
+use Codeception\Example;
+use hipanel\tests\_support\Step\Acceptance\Seller;
+use hipanel\tests\_support\Page\Widget\Input\Input;
+use hipanel\tests\_support\Page\Widget\Input\Select2;
+use hipanel\modules\finance\tests\_support\Page\transfer\Create as TransferCreate;
+use hipanel\modules\finance\tests\_support\Page\bill\Create as BillCreate;
+use hipanel\modules\finance\tests\_support\Page\transfer\Index;
+
+class InternalTransferCest
+{
+    private TransferCreate $transferCreate;
+    private BillCreate $billCreate;
+    private Index $index;
+
+    public function _before(Seller $I): void
+    {
+        $this->transferCreate = new TransferCreate($I);
+        $this->billCreate = new BillCreate($I);
+        $this->index = new Index($I);
+    }
+
+    /**
```

Comment:
```suggestion
    /**
     * IMPORTANT: this bill is needed because transfer can not be created when client has low balance
     * TODO: remove bill creation when transfer rules will be changed
```